eyeshade/workers/reports.js

```diff
@@ -98,6 +103,10 @@ const daily = async (debug, runtime) => {
       await runtime.queue.send(debug, 'publisher-report',
                                underscore.pick(entry, [ 'owner', 'publisher', 'verified', 'visible' ]))
     }
+
+    const surveyorsCollection = database.get('surveyors', debug)
+
+    await freezeOldSurveyors(7, midnight, surveyorsCollection)
```

Comment:
it might be good to make 7 configurable via env var

Reply:
that makes sense

Reply:
would you prefer this to default to 7? or should it err out if it is not given a date